libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-row-delete.directive.ts

```diff
@@ -158,19 +171,34 @@ export class SkyAgGridRowDeleteDirective
   #elementRef: ElementRef;
   #overlayService: SkyOverlayService;
   #viewContainerRef: ViewContainerRef;
+  #scrollableHostService: SkyScrollableHostService;
+  #clipPath$ = new BehaviorSubject<string | undefined>(undefined);
+  #zIndex = new BehaviorSubject('998');
```

Comment:
I'd be consistent with the variable names, either ending with a `$` or not. (My preference is without since variable names shouldn't hint at their types, but this might be something we need to decide on as a team. #flex)